src/IB/IIMethod.cpp

```diff
@@ -232,6 +232,126 @@ IIMethod::getFEDataManager(const unsigned int part) const
     TBOX_ASSERT(part < d_num_parts);
     return d_fe_data_managers[part];
 } // getFEDataManager
+NumericVector<double>* 
+IIMethod::getMeshCoordinatesNumeric(bool isCurrentConfiguration,std::string time,unsigned int part){
+
+
+    if(isCurrentConfiguration){
+
+        NumericVector<double>* X_ghost_vec = d_X_IB_ghost_vecs[part];
+        NumericVector<double>* X_vec = nullptr;
+
+        if(time == "current"){
+            X_vec = d_X_current_vecs[part];
+            copy_and_synch(*X_vec, *X_ghost_vec); 
+
+        }
+        else if(time == "half"){
+            X_vec = d_X_half_vecs[part];
+            copy_and_synch(*X_vec, *X_ghost_vec); 
+        }
+        else if(time == "new"){
+            X_vec = d_X_new_vecs[part];
+            copy_and_synch(*X_vec, *X_ghost_vec); 
+        }
+        else if(time == "ib_ghost"){
```

Comment:
`"ib_ghost"` is not a time...

Reply:
I am unsure at what time the `NumericVector<double>* X_ghost_vec = d_X_IB_ghost_vecs[part];` is getting the mesh at

Reply:
Should "time" be a different var name?